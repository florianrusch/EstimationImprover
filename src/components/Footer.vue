<template>
  <div
    class="
      w-full
      pt-16
      pb-6
      text-sm text-center
      md:text-left
      fade-in
      text-gray-500
    "
  >
    &copy; {{ year }} - Developed with ❤️ by Florian Rusch
  </div>
</template>

<script lang="ts">
import { defineComponent, ref } from "vue";

export default defineComponent({
  name: "Footer",
  setup() {
    var year = ref(new Date().getFullYear());

    return {
      year,
    };
  },
});
</script>
