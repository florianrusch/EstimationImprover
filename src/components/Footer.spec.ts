import { mount } from '@vue/test-utils'
import Footer from './Footer.vue'

describe('# Footer', () => {
	it('renders correctly', () => {
		// given
		const wrapper = mount(Footer)

		// when
		// then
		expect(wrapper.element).toMatchSnapshot()
	})
})
