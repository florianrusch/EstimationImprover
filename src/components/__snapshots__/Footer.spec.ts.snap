// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Footer renders correctly 1`] = `
<div
  class="w-full pt-16 pb-6 text-sm text-center md:text-left fade-in text-gray-500"
>
   © 2021 - Developed with ❤️ by Florian Rusch 
</div>
`;
